ui/src/views/Filter.vue

```diff
@@ -938,10 +938,19 @@ export default {
       this.filter[key] = this.filter[key] == "enabled" ? "disabled" : "enabled";
     },
     deleteRule(obj) {
-      this.filter.WBList.splice(this.filter.WBList.indexOf(obj), 1);
-      $("#deleteRuleModal").modal("hide");
+       var rulesindex = -1;
+       for(var i = 0; i < this.filter.WBList.length; i++){
+            var x = this.filter.WBList[i];
+            if(x['type'] == obj.type && x['value'] == obj.value){
+                rulesindex = i;
+                i = this.filter.WBList.length;
+            }
+        }
+        
+        this.filter.WBList.splice(rulesindex, 1);
+        $("#deleteRuleModal").modal("hide");
```

Comment:
This modification is not relevant to the describe issue.
Please remove it from the PR.